### DISCUSSION TITLE
How to connect server??

### DISCUSSION THREAD

**Discussion by klarsh** [Nimdy/Dedicated_Valheim_Server_Script#152] [Q&A] [answered]
//From South-Korea, so my ENG skills isn't good enough :(

First of all, thanks for your great source & script
I follow all your install steps, and did sudo ufw allow 2456:2458/tcp & /udp either.

After I install server and came back to main menu Server's Internet status shown as Disconnected.
I've already done Server Admin Tools -> Start Valheim Server but it doesn't work.

Can I get any advice Bro?

**Comment by Nimdy** [comment, marked as answer]
https://github.com/Nimdy/Dedicated_Valheim_Server_Script/wiki/Connecting-Manually-to-Valheim-Dedicated-Servers-through-Steam
